### PR TITLE
Makes a bunch of foods into finger foods

### PR DESCRIPTION
**Pull request by ArcaneDefence**
## About The Pull Request
Makes a bunch of things, such as sandwiches, slices of bread, snow cones, popsicles, donk pockets, chicken nuggets, candies, and cookies to name some into finger foods, allowing you to eat them while walking.

## Why It's Good For The Game
Dethrones donuts as the king of snacking on the go
Reinforces my immersion as I no longer need to stop in place to chomp down on a slice of bread

## Changelog
:cl:
add: Some foods from sandwiches to donk pockets to candies and snowcones can now be eaten while walking.
/:cl:

### CODE REVIEW

**Comment by MrDoomBringer** [comment]
>donk pockets

nice omnizine combat healing buff

**Comment by ArcaneDefence** [comment]
> 
> nice omnizine combat healing buff

image
All I'm hearing is get rid of omnizine from my favorite microwavable snack that we give out for free, to just about everyone.

**Comment by Time-Green** [comment]
Wait I could’ve moved while eating donuts all this time?!?.

**Comment by maxymax13** [comment]
sounds like someone got killed by an assistant eating donk pockets

File: code/game/objects/items/food/bread.dm [hunk 1]
```diff
@@ -15,6 +15,7 @@
 	icon = 'icons/obj/food/burgerbread.dmi'
 	slot_flags = ITEM_SLOT_HEAD
 	foodtypes = GRAIN
+	food_flags = FOOD_FINGER_FOOD
```
Comment (Qustinnus) [inline code/game/objects/items/food/bread.dm:18]: what is this? a japanese anime?

**Comment by Mothblocks** [comment]
> sounds like someone got killed by an assistant eating donk pockets

i did

**Pull request merged**